### ISSUE TITLE
error "The kernel appears to have died. It will restart automatically"  when import rdkit in the jupyter notebook

### ISSUE THREAD

**Issue by AustinApple** [rdkit/rdkit#2065]
**Description:**
When I import rdkit in the jupyter notebook, the error pops out a few days ago. However, it work normally before this. But, I also found that it work normally on the terminal. Is anyone suffer this problem?

- RDKit Version: 2018.03.3
- Platform: jupyter notebook 

```
Your code sample herefrom rdkit.Chem import AllChem
from rdkit import Chem
from rdkit.Chem import Draw
import pandas 
data_smile = pandas.read_csv('test.csv',usecols=[0],header=0)

dict = {}   
def ECFP(row):
    try:
        m = Chem.MolFromSmiles(row['smiles'])
        print(row['smiles']+"is succeed")
        info={}
        fp = AllChem.GetMorganFingerprintAsBitVect(m,2,nBits=512,bitInfo=info)
        print(info)
        for bit in info:
            for i in info[bit]:
                env = Chem.FindAtomEnvironmentOfRadiusN(m,i[1],i[0])
                amap={}
                submol=Chem.PathToSubmol(m,env,atomMap=amap)
                smile = Chem.MolToSmiles(submol)
                dict[bit] = smile
    except:
        print(row['smiles']+"fails!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!")
    
            
data_smile = data_smile.apply(ECFP, axis=1)
```

**Comment by greglandrum** [comment]
Are you sure that the jupyter you are using is the one that's installed in the conda environment where the rdkit is installed?

**Comment by greglandrum** [comment]
I’m somewhat surprised by that; I would expect 3.6.5 to work.
Which operating system are you on?

**Comment by AustinApple** [comment]
macOS High Sierra 10.13.6（17G65）

**Comment by somnambWl** [comment]
I have the same problem, but it is not solved with different Python version, I have tried (3.5.5, 3.6.7 and 3.6.8). Kernel keeps restarting without an error when I run RDKit.

**Comment by greglandrum** [comment]
@somnambWl: Without more info, there's nothing we can do to help here. Which operating system and version of conda are you using?
Please also send the output of `conda list` so that we can see which packages are installed.

**Comment by MashaAseeva** [comment]
Hello, I am also getting this error. Here is the output of my conda-list:
`# Name                    Version                   Build  Channel
appnope                   0.1.0                    py35_0    conda-forge
backcall                  0.1.0                      py_0    conda-forge
blas                      1.0                         mkl  
bleach                    3.1.0                      py_0    conda-forge
boost                     1.66.0                   py35_1    conda-forge
boost-cpp                 1.66.0                        1    conda-forge
bzip2                     1.0.8                h1de35cc_0  
ca-certificates           2019.11.28           hecc5488_0    conda-forge
cairo                     1.14.12              hc4e6be7_4  
certifi                   2018.8.24             py35_1001    conda-forge
cirpy                     1.0.2                    pypi_0    pypi
decorator                 4.4.1                      py_0    conda-forge
defusedxml                0.6.0                      py_0    conda-forge
entrypoints               0.2.3                    py35_2    conda-forge
fontconfig                2.13.0               h5d5b041_1  
freetype                  2.9.1                hb4e5f40_0  
gettext                   0.19.8.1             h15daf44_3  
glib                      2.63.1               hd977a24_0  
icu                       58.2                 h4b95b61_1  
intel-openmp              2019.4                      233  
ipykernel                 5.1.0              pyh24bf2e0_0    conda-forge
ipython                   7.0.1            py35h24bf2e0_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
jedi                      0.12.1                   py35_0    conda-forge
jinja2                    2.11.0                     py_0    conda-forge
jpeg                      9b                   he5867d9_2  
json5                     0.8.5                      py_0    conda-forge
jsonschema                2.6.0                    py35_2    conda-forge
jupyter_client            5.3.3                      py_0    conda-forge
jupyter_core              4.5.0                      py_0    conda-forge
jupyterlab                1.2.6                      py_0    conda-forge
jupyterlab_server         1.0.0                      py_0  
libcxx                    4.0.1                hcfea43d_1  
libcxxabi                 4.0.1                hcfea43d_1  
libedit                   3.1.20181209         hb402a30_0  
libffi                    3.2.1                         1    bioconda
libgfortran               3.0.1                h93005f0_2  
libiconv                  1.15                 hdd342a3_7  
libpng                    1.6.37               ha441bb4_0  
libsodium                 1.0.18               h01d97ff_0    conda-forge
libtiff                   4.1.0                hcb84e12_0  
libxml2                   2.9.9                hf6e021a_1  
libxslt                   1.1.33               h33a18ac_0  
lxml                      4.2.5            py35hef8c89e_0  
markupsafe                1.0              py35h470a237_1    conda-forge
mistune                   0.8.3            py35h470a237_2    conda-forge
mkl                       2018.0.3                      1  
mkl_fft                   1.0.6            py35hb8a8100_0  
mkl_random                1.0.1            py35h5d10147_1  
nbconvert                 5.6.0                      py_0    conda-forge
nbformat                  5.0.4                      py_0    conda-forge
ncurses                   6.1                  h0a44026_1  
notebook                  5.7.0                    py35_0    conda-forge
numpy                     1.15.2           py35h6a91979_0  
numpy-base                1.15.2           py35h8a80b8c_0  
olefile                   0.46                       py_0  
openssl                   1.0.2u               h0b31af3_0    conda-forge
pandas                    0.23.4           py35h6440ff4_0  
pandoc                    2.9.1.1                       0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
parso                     0.6.0                      py_0    conda-forge
pcre                      8.43                 h0a44026_0  
pexpect                   4.6.0                    py35_0    conda-forge
pickleshare               0.7.5                    py35_0    conda-forge
pillow                    5.2.0            py35hb68e598_0  
pip                       10.0.1                   py35_0  
pixman                    0.38.0               h1de35cc_0  
prometheus_client         0.7.1                      py_0    conda-forge
prompt_toolkit            2.0.10                     py_0    conda-forge
ptyprocess                0.6.0                   py_1001    conda-forge
pycairo                   1.17.1           py35ha54c0a8_0  
pygments                  2.5.2                      py_0    conda-forge
python                    3.5.6                hc167b69_0  
python-dateutil           2.8.1                      py_0    conda-forge
pytz                      2019.3                     py_0  
pyzmq                     17.1.2           py35hae99301_0    conda-forge
rdkit                     2018.03.4        py35h557c172_0    conda-forge
readline                  7.0                  h1de35cc_5  
send2trash                1.5.0                      py_0    conda-forge
setuptools                40.2.0                   py35_0  
simplegeneric             0.8.1                      py_1    conda-forge
six                       1.11.0                   py35_1    conda-forge
sqlite                    3.30.1               ha441bb4_0  
tbb                       2019.8               h04f5b5a_0  
tbb4py                    2018.0.5         py35h04f5b5a_0  
terminado                 0.8.1                    py35_1    conda-forge
testpath                  0.4.4                      py_0    conda-forge
tk                        8.6.8                ha441bb4_0  
tornado                   5.1.1            py35h470a237_0    conda-forge
traitlets                 4.3.2                    py35_0    conda-forge
wcwidth                   0.1.8                      py_0    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.31.1                   py35_0  
xlrd                      1.2.0                      py_0  
xz                        5.2.4                h1de35cc_4  
zeromq                    4.2.5                hfc679d8_4    conda-forge
zlib                      1.2.11               h1de35cc_3  
zstd                      1.3.7                h5bba6e5_0 `

**Comment by wesleymsmith** [comment]
I am getting the same error.

I am running on a iMac workstation 

macOS Mojave (version 10.14.6)
Intel Core i5 3.7 GHz processor

python info:

```
python
Python 3.7.6 | packaged by conda-forge | (default, Jan  7 2020, 22:05:27)
[Clang 9.0.1 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
```

Here is my conda package list:

```
conda list
# packages in environment at /Users/wbotellosmith/anaconda2/envs/cheminfo:
#
# Name                    Version                   Build  Channel
alabaster                 0.7.12                     py_0    conda-forge
appnope                   0.1.0                    py37_0    anaconda
attrs                     19.3.0                     py_0    anaconda
babel                     2.8.0                      py_0    conda-forge
backcall                  0.1.0                    py37_0    anaconda
bel-resources             0.0.3                      py_0    bioconda
blas                      2.15                   openblas    conda-forge
bleach                    3.1.0                    py37_0    anaconda
boost                     1.68.0          py37h9888f84_1001    conda-forge
boost-cpp                 1.68.0            h6f8c590_1000    conda-forge
bzip2                     1.0.8                h0b31af3_2    conda-forge
ca-certificates           2020.1.1                      0    anaconda
cairo                     1.14.12           h9d4d9ac_1005    conda-forge
certifi                   2019.11.28               py37_0    anaconda
cffi                      1.13.2           py37h33e799b_0    conda-forge
chardet                   3.0.4                 py37_1003    conda-forge
click                     7.0                        py_0    conda-forge
click-plugins             1.1.1                      py_0    conda-forge
cryptography              2.8              py37hafa8578_1    conda-forge
cycler                    0.10.0                   py37_0    anaconda
dbus                      1.13.12              h90a0687_0    anaconda
decorator                 4.4.1                      py_0    anaconda
defusedxml                0.6.0                      py_0    anaconda
docutils                  0.16                     py37_0    conda-forge
entrypoints               0.3                      py37_0    anaconda
expat                     2.2.6                h0a44026_0    anaconda
fontconfig                2.13.1            h1027ab8_1000    conda-forge
freetype                  2.10.0               h24853df_1    conda-forge
gettext                   0.19.8.1             h15daf44_3    anaconda
glib                      2.56.2               hd9629dc_0    anaconda
icu                       58.2                 h4b95b61_1    anaconda
idna                      2.9                        py_1    conda-forge
imagesize                 1.2.0                      py_0    conda-forge
importlib_metadata        1.5.0                    py37_0    anaconda
ipykernel                 5.1.4            py37h39e3cac_0    anaconda
ipython                   7.12.0           py37h5ca1d4c_0    anaconda
ipython_genutils          0.2.0                    py37_0    anaconda
ipywidgets                7.5.1                      py_0    anaconda
jedi                      0.16.0                   py37_0    anaconda
jinja2                    2.11.1                     py_0    anaconda
joblib                    0.14.1                     py_0    anaconda
jpeg                      9b                   he5867d9_2    anaconda
jsonschema                3.2.0                    py37_0    anaconda
jupyter                   1.0.0                    py37_7    anaconda
jupyter_client            5.3.4                    py37_0    anaconda
jupyter_console           6.1.0                      py_0    anaconda
jupyter_core              4.6.1                    py37_0    anaconda
kiwisolver                1.1.0            py37h0a44026_0    anaconda
libblas                   3.8.0               15_openblas    conda-forge
libcblas                  3.8.0               15_openblas    conda-forge
libcxx                    9.0.1                         1    conda-forge
libffi                    3.2.1             h6de7cb9_1006    conda-forge
libgfortran               4.0.0                         2    conda-forge
libiconv                  1.15                 hdd342a3_7    anaconda
liblapack                 3.8.0               15_openblas    conda-forge
liblapacke                3.8.0               15_openblas    conda-forge
libopenblas               0.3.8                h3d69b6c_0    conda-forge
libpng                    1.6.37               ha441bb4_0    anaconda
libsodium                 1.0.16               h3efe00b_0    anaconda
libtiff                   4.0.9                he6b73bb_1    conda-forge
libxml2                   2.9.9                hd80cff7_2    conda-forge
llvm-openmp               9.0.1                h28b9765_2    conda-forge
markupsafe                1.1.1            py37h1de35cc_0    anaconda
matplotlib                3.1.3                    py37_0    anaconda
matplotlib-base           3.1.3            py37h9aa3819_0    anaconda
mistune                   0.8.4            py37h1de35cc_0    anaconda
multisplitby              0.0.1                      py_0    conda-forge
nbconvert                 5.6.1                    py37_0    anaconda
nbformat                  5.0.4                      py_0    anaconda
ncurses                   6.1               h0a44026_1002    conda-forge
networkx                  2.4                        py_0    conda-forge
notebook                  6.0.3                    py37_0    anaconda
numpy                     1.18.1           py37hde6bac1_0    conda-forge
numpydoc                  0.9.2                      py_0    conda-forge
oddt                      0.7                        py_0    oddt
olefile                   0.46                       py_0    conda-forge
openbabel                 2.4.1                    py37_6    openbabel
openssl                   1.1.1                h1de35cc_0    anaconda
packaging                 20.1                       py_0    conda-forge
pandas                    1.0.1            py37h4f17bb1_0    conda-forge
pandoc                    2.2.3.2                       0    anaconda
pandocfilters             1.4.2                    py37_1    anaconda
parso                     0.6.1                      py_0    anaconda
pcre                      8.43                 h0a44026_0    anaconda
pexpect                   4.8.0                    py37_0    anaconda
pickleshare               0.7.5                    py37_0    anaconda
pillow                    5.4.1            py37hb68e598_0
pip                       20.0.2                     py_2    conda-forge
pixman                    0.34.0            h1de35cc_1003    conda-forge
prometheus_client         0.7.1                      py_0    anaconda
prompt_toolkit            3.0.3                      py_0    anaconda
ptyprocess                0.6.0                    py37_0    anaconda
pubchempy                 1.0.4                      py_1    bioconda
pybel                     0.13.2                     py_0    bioconda
pycairo                   1.18.0          py37ha54c0a8_1000    conda-forge
pycparser                 2.19                       py_2    conda-forge
pygments                  2.5.2                      py_0    anaconda
pyopenssl                 19.1.0                     py_1    conda-forge
pyparsing                 2.4.6                      py_0    conda-forge
pyqt                      5.9.2            py37h655552a_0    anaconda
pyrsistent                0.15.7           py37h1de35cc_0    anaconda
pysocks                   1.7.1                    py37_0    conda-forge
python                    3.7.6                h5c2c468_2    conda-forge
python-dateutil           2.8.1                      py_0    anaconda
pytz                      2019.3                     py_0    conda-forge
pyzmq                     18.1.1           py37h0a44026_0    anaconda
qt                        5.9.7                h468cd18_1    anaconda
qtconsole                 4.6.0                      py_1    anaconda
rdkit                     2018.09.1       py37hdb4e85a_1000    conda-forge
readline                  8.0                  hcfe32e1_0    conda-forge
requests                  2.23.0                   py37_0    conda-forge
requests-file             1.4.3                      py_0    conda-forge
scikit-learn              0.22.1           py37hebd9d1a_0    anaconda
scipy                     1.4.1            py37h82752d6_0    conda-forge
seaborn                   0.10.0                     py_0    anaconda
send2trash                1.5.0                    py37_0    anaconda
setuptools                45.2.0                   py37_0    conda-forge
sip                       4.19.13          py37h0a44026_0    anaconda
six                       1.14.0                   py37_0    anaconda
snowballstemmer           2.0.0                      py_0    conda-forge
sphinx                    2.4.3                      py_0    conda-forge
sphinxcontrib-applehelp   1.0.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.2                      py_0    conda-forge
sphinxcontrib-htmlhelp    1.0.3                      py_0    conda-forge
sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge
sphinxcontrib-qthelp      1.0.3                      py_0    conda-forge
sphinxcontrib-serializinghtml 1.1.4                      py_0    conda-forge
sqlalchemy                1.3.13           py37h0b31af3_0    conda-forge
sqlite                    3.30.1               h93121df_0    conda-forge
terminado                 0.8.3                    py37_0    anaconda
testpath                  0.4.4                      py_0    anaconda
tk                        8.6.10               hbbe82c9_0    conda-forge
tornado                   6.0.3            py37h1de35cc_3    anaconda
tqdm                      4.43.0                     py_0    conda-forge
traitlets                 4.3.3                    py37_0    anaconda
urllib3                   1.25.7                   py37_0    conda-forge
wcwidth                   0.1.8                      py_0    anaconda
webencodings              0.5.1                    py37_1    anaconda
wheel                     0.34.2                     py_1    conda-forge
widgetsnbextension        3.5.1                    py37_0    anaconda
xz                        5.2.4             h1de35cc_1001    conda-forge
zeromq                    4.3.1                h0a44026_3    anaconda
zipp                      2.2.0                      py_0    anaconda
zlib                      1.2.11            h0b31af3_1006    conda-forge
```

**Comment by ilymak** [comment]
I was getting this error after I've installed rdkit in an existing conda environment as described in the rdkit [docs](https://www.rdkit.org/docs/Install.html): `conda install -c conda-forge rdkit`

However, if I create a new conda environment with a command (also from the [docs](https://www.rdkit.org/docs/Install.html)): `conda create -c rdkit -n my-rdkit-env rdkit` then the error does not appear and I can import and use rdkit in my jupyter notebook. 

It looks like the error was caused by some conflicting dependencies.

**Comment by JinChengneng** [comment]
I am stuck in the same error. I am using jupyter notebook on Safari with macOS 10.15.6.

Once I execute ```mol = Chem.MolFromSmiles(smiles)```, the kernel will die without any errors from  ```rdkit```.

The following is my conda list.
```
# packages in environment at /Users/dip/anaconda3/envs/py3.7:
#
# Name                    Version                   Build  Channel
_pytorch_select           0.1                       cpu_0    defaults
_tflow_select             2.3.0                       mkl    defaults
absl-py                   0.10.0           py37hc8dfbb8_0    conda-forge
appnope                   0.1.0           py37hc8dfbb8_1001    conda-forge
argon2-cffi               20.1.0           py37h60d8a13_1    conda-forge
astor                     0.8.1              pyh9f0ad1d_0    conda-forge
astroid                   2.4.2                    py37_0    defaults
attrs                     20.1.0             pyh9f0ad1d_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
blas                      1.0                         mkl    defaults
bleach                    3.1.0                    py37_0    defaults
boost                     1.70.0           py37hbf1eeb5_1    conda-forge
boost-cpp                 1.70.0               hd59e818_0    conda-forge
bzip2                     1.0.6                         3    defaults
c-ares                    1.16.1               haf1e3a3_3    conda-forge
ca-certificates           2020.7.22                     0    defaults
cairo                     1.16.0            h0ab9d94_1001    conda-forge
certifi                   2020.6.20                py37_0    defaults
cffi                      1.14.2           py37hed5b41f_0    defaults
cycler                    0.10.0                     py_2    conda-forge
dbus                      1.13.6               h2f22bb5_0    conda-forge
decorator                 4.4.2                      py_0    conda-forge
defusedxml                0.6.0                      py_0    conda-forge
entrypoints               0.3             py37hc8dfbb8_1001    conda-forge
expat                     2.2.9                h4a8c4bd_2    conda-forge
fontconfig                2.13.1            h1027ab8_1000    conda-forge
freetype                  2.10.2               ha233b18_0    defaults
gast                      0.2.2                      py_0    conda-forge
gettext                   0.19.8                        1    defaults
glib                      2.65.0               hc5f4afa_0    defaults
google-pasta              0.2.0              pyh8c360ce_0    conda-forge
grpcio                    1.30.0           py37hf940fad_0    conda-forge
h5py                      2.10.0          nompi_py37h106b333_102    conda-forge
hdf5                      1.10.5          nompi_h0cbb7df_1103    conda-forge
icu                       58.2              h0a44026_1000    conda-forge
importlib-metadata        1.7.0            py37hc8dfbb8_0    conda-forge
importlib_metadata        1.7.0                         0    conda-forge
intel-openmp              2019.4                      233    defaults
ipykernel                 5.3.4            py37h43977f1_0    conda-forge
ipython                   7.18.1           py37hc6149b9_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.5.1                      py_0    conda-forge
isort                     5.4.2                    py37_0    defaults
jedi                      0.15.2                   py37_0    conda-forge
jinja2                    2.11.2             pyh9f0ad1d_0    conda-forge
joblib                    0.16.0                     py_0    conda-forge
jpeg                      9b                   he5867d9_2    defaults
jsonschema                3.2.0            py37hc8dfbb8_1    conda-forge
jupyter                   1.0.0                      py_2    conda-forge
jupyter_client            6.1.7                      py_0    conda-forge
jupyter_console           6.2.0                      py_0    conda-forge
jupyter_core              4.6.3            py37hc8dfbb8_1    conda-forge
keras-applications        1.0.8                      py_1    conda-forge
keras-preprocessing       1.1.0                      py_0    conda-forge
kiwisolver                1.2.0            py37ha1cc60f_0    conda-forge
lazy-object-proxy         1.4.3            py37h1de35cc_0    defaults
lcms2                     2.11                 h92f6f08_0    defaults
libblas                   3.8.0                    14_mkl    conda-forge
libcblas                  3.8.0                    14_mkl    conda-forge
libcxx                    10.0.0                        1    defaults
libedit                   3.1.20191231         h1de35cc_1    defaults
libffi                    3.3                  hb1e8313_2    defaults
libgfortran               3.0.1                         0    conda-forge
libiconv                  1.16                 haf1e3a3_0    conda-forge
libpng                    1.6.37               ha441bb4_0    defaults
libprotobuf               3.12.3               hab81aa3_2    conda-forge
libsodium                 1.0.17               h01d97ff_0    conda-forge
libtiff                   4.1.0                hcb84e12_1    defaults
libxml2                   2.9.9                hd80cff7_2    conda-forge
llvm-openmp               10.0.1               h28b9765_0    conda-forge
lz4-c                     1.9.2                hb1e8313_1    defaults
markdown                  3.2.2                      py_0    conda-forge
markupsafe                1.1.1            py37h9bfed18_1    conda-forge
matplotlib                3.3.1                         1    conda-forge
matplotlib-base           3.3.1            py37h181983e_0    defaults
mccabe                    0.6.1                    py37_1    defaults
mistune                   0.8.4           py37h9bfed18_1001    conda-forge
mkl                       2019.4                      233    defaults
mkl-service               2.3.0            py37hfbe908c_0    defaults
mkl_fft                   1.1.0            py37hc64f4ea_0    defaults
mkl_random                1.1.1            py37h959d312_0    defaults
nbconvert                 5.6.1            py37hc8dfbb8_1    conda-forge
nbformat                  5.0.7                      py_0    conda-forge
ncurses                   6.2                  h0a44026_1    defaults
ninja                     1.10.0           py37h879752b_0    defaults
notebook                  6.1.3            py37hc8dfbb8_0    conda-forge
numpy                     1.19.1           py37h3b9f5b6_0    defaults
numpy-base                1.19.1           py37hcfb5961_0    defaults
olefile                   0.46                     py37_0    defaults
openssl                   1.1.1g               h1de35cc_0    defaults
opt_einsum                3.3.0                      py_0    conda-forge
pandas                    1.0.5            py37h94625e5_0    conda-forge
pandoc                    1.19.2.1                      0    defaults
pandocfilters             1.4.2                      py_1    conda-forge
parso                     0.5.2                      py_0    defaults
pcre                      8.44                 h4a8c4bd_0    conda-forge
pexpect                   4.8.0            py37hc8dfbb8_1    conda-forge
pickleshare               0.7.5           py37hc8dfbb8_1001    conda-forge
pillow                    7.2.0            py37ha54b6ba_0    defaults
pip                       20.2.2                   py37_0    defaults
pixman                    0.38.0            h01d97ff_1003    conda-forge
prometheus_client         0.8.0              pyh9f0ad1d_0    conda-forge
prompt-toolkit            3.0.7                      py_0    conda-forge
prompt_toolkit            3.0.7                         0    conda-forge
protobuf                  3.12.3           py37h570ac47_0    conda-forge
ptyprocess                0.6.0                   py_1001    conda-forge
pycairo                   1.19.1           py37h4e38901_3    conda-forge
pycparser                 2.20                       py_2    defaults
pygments                  2.6.1                      py_0    conda-forge
pylint                    2.6.0                    py37_0    defaults
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyqt                      5.9.2            py37h2a560b1_4    conda-forge
pyrsistent                0.16.0           py37h9bfed18_0    conda-forge
python                    3.7.7                hf48f09d_4    defaults
python-dateutil           2.8.1                      py_0    conda-forge
python_abi                3.7                     1_cp37m    conda-forge
pytorch                   1.3.1           cpu_py37h0c87eb2_0    defaults
pytz                      2020.1             pyh9f0ad1d_0    conda-forge
pyzmq                     18.1.1           py37h4bf09a9_0    conda-forge
qt                        5.9.7                h468cd18_1    defaults
qtconsole                 4.7.6              pyh9f0ad1d_0    conda-forge
qtpy                      1.9.0                      py_0    conda-forge
rdkit                     2019.09.3        py37h3471270_0    conda-forge
readline                  8.0                  h1de35cc_0    defaults
scikit-learn              0.21.3           py37hd4ffd6c_0    conda-forge
scipy                     1.5.2            py37h912ce22_0    defaults
send2trash                1.5.0                      py_0    conda-forge
setuptools                49.6.0                   py37_0    defaults
sip                       4.19.8           py37h0a44026_0    defaults
six                       1.15.0                     py_0    defaults
sqlite                    3.33.0               hffcf06c_0    defaults
tensorboard               2.0.0              pyhb38c66f_1    defaults
tensorflow                2.0.0           mkl_py37hda344b4_0    defaults
tensorflow-base           2.0.0           mkl_py37h66b1bf0_0    defaults
tensorflow-estimator      2.0.0              pyh2649769_0    defaults
termcolor                 1.1.0                      py_2    conda-forge
terminado                 0.8.3            py37hc8dfbb8_1    conda-forge
testpath                  0.4.4                      py_0    conda-forge
tk                        8.6.10               hb0a8c7a_0    defaults
toml                      0.10.1                     py_0    defaults
torchvision               0.4.2           cpu_py37h1ea6fa9_0    defaults
tornado                   6.0.4            py37h9bfed18_1    conda-forge
tqdm                      4.48.2                     py_0    defaults
traitlets                 4.3.3            py37hc8dfbb8_1    conda-forge
typed-ast                 1.4.1            py37h1de35cc_0    defaults
wcwidth                   0.2.5              pyh9f0ad1d_0    conda-forge
webencodings              0.5.1                      py_1    conda-forge
werkzeug                  1.0.1              pyh9f0ad1d_0    conda-forge
wheel                     0.35.1                     py_0    defaults
widgetsnbextension        3.5.1            py37hc8dfbb8_1    conda-forge
wrapt                     1.11.2           py37h1de35cc_0    defaults
xz                        5.2.5                h1de35cc_0    defaults
zeromq                    4.3.2                h6de7cb9_2    conda-forge
zipp                      3.1.0                      py_0    conda-forge
zlib                      1.2.11               h1de35cc_3    defaults
zstd                      1.4.5                h41d2c2f_0    defaults
```

**Comment by MichaelKnight4714** [comment]
Did anyone ever solve this issue? I tried "conda create -c rdkit -n my-rdkit-env rdkit", but still nothing... is there something i am supposed to do before that command, or after? Please help

**Comment by greglandrum** [comment]
@MichaelKnight4714 : please install the rdkit from conda-forge; we don't keep the builds in the rdkit channel up to date.
If you still have the problem after doing that, please paste in the contents of your conda environment and let us know which operating system you're using

**Comment by MichaelKnight4714** [comment]
> @MichaelKnight4714 : please install the rdkit from conda-forge; we don't keep the builds in the rdkit channel up to date. If you still have the problem after doing that, please paste in the contents of your conda environment and let us know which operating system you're using

@greglandrum ok how? I installed it once before, and I forked and cloned the mmltoolkit folder (as well as the QM9_data_import_and_processes folder), and am now trying to run `conda install -c conda-forge rdkit` . I get the error message:
"Collecting package metadata (repodata.json): failed

UnavailableInvalidChannel: The channel is not accessible or is invalid.
  channel name: mmltoolkit
  channel url: https://conda.anaconda.org/mmltoolkit
  error code: 404

You will need to adjust your conda configuration to proceed.
Use `conda config --show channels` to view your configuration's current state,
and use `conda config --show-sources` to view config file locations."

Any advice?
